service/src/main/java/bio/terra/externalcreds/dataAccess/EncryptDecryptUtils.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.externalcreds.dataAccess;
+
+import com.google.cloud.kms.v1.CryptoKeyName;
+import com.google.cloud.kms.v1.DecryptResponse;
+import com.google.cloud.kms.v1.EncryptResponse;
+import com.google.cloud.kms.v1.KeyManagementServiceClient;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+
+/** Utils for KMS symmetric encryption and decryption. */
+public class EncryptDecryptUtils {
+
+  private EncryptDecryptUtils() {}
+
+  /** Encrypt with KMS symmetric key. */
+  public static String encryptSymmetric(
+      String projectId, String locationId, String keyRingId, String keyId, String plainText)
+      throws IOException {
+    try (var client = KeyManagementServiceClient.create()) {
+      var keyVersionName = CryptoKeyName.of(projectId, locationId, keyRingId, keyId);
+      EncryptResponse response = client.encrypt(keyVersionName, ByteString.copyFromUtf8(plainText));
+      client.close();
```

Comment:
isn't the close implicit in the try?

Reply:
Yep - btw @yuhuyoyo if you look at SonarCloud code quality output, it suggest the same.